common/changes/@snowplow/browser-plugin-accelerator-ecommerce/feature-ecommerce-accelerator-browser_2022-09-28-11-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/browser-plugin-accelerator-ecommerce",
```

Comment:
You might have already discussed and pondered this, so ignore me if so, but the naming of the plugin raises two small concerns for me:

1. Should we tie the plugin to the accelerator? In my view, the plugin is more widely usable, also outside of the accelerator. Using accelerator in the name seems like constraining the plugin a bit to me. For some users, it might just be the ecommerce plugin that they use in their apps long term, not just when they were learning through an accelerator, right?
2. We already have two ecommerce plugins (ecommerce and enhanced-ecommerce). Adding another makes this even more confusing for users. Could we instead have this be the next major version of the ecommerce or enhanced ecommerce plugins?

Reply:
I agree, I don't want to couple this to the accelerator, but I also don't really fancy calling it `browser-plugin-really-enhanced-ecommerce`.

Maybe `browser-plugin-retail` (? also begs the question if we call this ecommerce or retail as it perhap has broader applications than _just_ ecommerce). This can go out in a `v3` release, where we also mark the other two as deprecated. Then in v4 we remove them and add this new one into sp.js (which is a big move but one I think we can navigate with messaging if this launches in a v3).